ISPy/spec/crosstalk.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+
+
+# ========================================================================
+def get_projection(u,v):
+    """It outputs the scalar projection using the inner product of the vectors.
+    
+    Parameters
+    ----------
+    u, v : ndarray
+        input vectors
+    
+    Returns
+    -------
+    float
+        Scalar projection
+    """
+
+    return np.inner(u,v)/np.inner(v,v)
+
+# ========================================================================
+def estimate_crosstalk(stokes_map,stokes_toclean, stokes_removefrom, nameoutput=None, interactive=True, 
+    npoints=5, verbose=True, sizeave=3, smoothvalue=100):
+    """Given the input cube it outputs an estimation of the crosstalk between them 
+    for one time frame.
+    
+    Parameters
+    ----------
+    stokes_map : ndarray
+        Data cube for one time frame
+    stokes_toclean : int
+        Stokes index to clean (usually 1 or 2)
+    stokes_removefrom : int
+        Stokes index to clean (usually 3)
+    nameoutput : None, optional
+        Name of the output file with the field-dependent crosstalk and figures
+    interactive : bool, optional
+        Interactive method where the user selects points in the FOV.
+    npoints : int, optional
+        Number of points to be selected
+    verbose : bool, optional
+        It prints the average crosstalk calculated with both methods
+    sizeave : int, optional
+        Window size to average after clicking
+    smoothvalue : int, optional
+        Smoothing width of the kernel after extrapolation
+    
+    Returns
+    -------
+    ndarray
+        Field-dependent crosstalk if interactive is True or a the mean value calculated by the automatic method if False
+       
+    Example
+    -------
+    >>> estimate_crosstalk(data[0,:], stokes_toclean=1, stokes_removefrom=3, nameoutput='Q_test')
+    
+    :Authors: 
+        Carlos Diaz (ISP/SU 2019)
+
+    """
+    stokes_label = ['I','Q','U','V']
+    mapa_coef = np.zeros((stokes_map.shape[2],stokes_map.shape[3]))
```

Comment:
Better use an English variable name. ;) Similarly elsewhere in the code (e.g. lines 64-65 `datos_from` and `datos_to`).